metrics/configs/presubmit-health.yaml

```diff
@@ -0,0 +1,91 @@
+metric: presubmit-health
+description: Calculate daily health metrics for presubmits
+query: |
+  #standardSQL
+  SELECT
+    UNIX_SECONDS(day) day,
+    job,
+    ROUND(prs_failed/prs * 100, 1) pr_failure_perc,
+    prs,
+    prs_failed,
+    total_runs total_runs,
```

Comment:
nit: You only need one `total_runs` here.

Reply:
fixed